web/src/components/organisms/Settings/Project/Dataset/hooks.ts

```diff
@@ -106,6 +109,33 @@ export default (projectId: string) => {
     [client, importData, sceneId],
   );
 
+  //Download file
+
+  const handleDownloadFile = useCallback(
+    async (id: string, name: string) => {
+      if (!id || !window.REEARTH_CONFIG?.api) return;
+
+      const accessToken = await getAccessToken();
+      if (!accessToken) return;
+
+      const res = await fetch(`${window.REEARTH_CONFIG.api}/datasets/${id}`, {
+        headers: {
+          ...(accessToken ? { Authorization: `Bearer ${accessToken}` } : {}),
+        },
+      });
+      const download = document.createElement("a");
+      download.download = name;
+      download.href = URL.createObjectURL(await res.blob());
+      download.dataset.downloadurl = [
+        "data:text/csv;charset=utf-8,",
+        download.download,
+        download.href,
+      ].join(":");
```

Comment:
Why is this necessary?

Reply:
Without this part of the code, the download element would not have a valid downloadurl attribute.
Therefore, it is necessary to set the downloadurl attribute in order to ensure that the CSV file is downloaded correctly when the user clicks on the download link. do you want me to remove it?

Reply:
Hmm, I can't find specification for `data-downloadurl` attribute. I think it should remove if it works even if it is removed.

Reply:
Done